src/main/scala/org/dbpedia/spotlight/wikistats/JsonPediaParser.scala

```diff
@@ -135,16 +135,21 @@ class JsonPediaParser(inputWikiDump: String, lang: String)
     Input:  - None
     Output: - RDD of all wikiId and the article text
    */
-  def getArticleText(): RDD[(Long, String)] = {
 
-    dfWikiRDD.select("wid","wikiText","type")
+  def getArticleText(): RDD[(Long, String, List[(String, Long, String)])] = {
```

Comment:
I think creating a case class`Article` containing all of those fields would make the code more readable.

Consider code using this method, doing `result._3.(10)` for example it is very obscure. as opposed to:
`result.links(10)`

It is